lib/internal/modules/cjs/loader.js

```diff
@@ -451,15 +451,17 @@ function tryPackage(requestPath, exts, isMain, originalPath) {
       err.code = 'MODULE_NOT_FOUND';
       err.path = path.resolve(requestPath, 'package.json');
       err.requestPath = originalPath;
-      // TODO(BridgeAR): Add the requireStack as well.
+
       throw err;
     } else {
+      const requireStack = [];
       const jsonPath = path.resolve(requestPath, 'package.json');
       process.emitWarning(
         `Invalid 'main' field in '${jsonPath}' of '${pkg}'. ` +
           'Please either fix that or report it to the module author',
         'DeprecationWarning',
         'DEP0128',
+        { requireStack },
```

Comment:
I am not certain what the intent here is. The warning seems independent from the TODO entry above?